build-ps/percona-server-5.7_builder.sh

```diff
@@ -716,14 +716,26 @@ build_tarball(){
     export CXXFLAGS="${CFLAGS}"
     if [ "${YASSL}" = 0 ]; then
         if [ -f /etc/redhat-release ]; then
-            SSL_VER_TMP=$(yum list installed|grep -i openssl|head -n1|awk '{print $2}'|awk -F "-" '{print $1}'|sed 's/\.//g'|sed 's/[a-z]$//')
+            if [ "${RHEL}" -eq 8 ]; then
+                SSL_VER_TMP=$(yum list installed|grep -i openssl|head -n1|awk '{print $2}'|awk -F ":" '{print $2}'|awk -F "-" '{print $1}'|sed 's/\.//g'|sed 's/[a-z]$//')
+            else
+                SSL_VER_TMP=$(yum list installed|grep -i openssl|head -n1|awk '{print $2}'|awk -F "-" '{print $1}'|sed 's/\.//g'|sed 's/[a-z]$//')
+            fi
             export SSL_VER=".ssl${SSL_VER_TMP}"
```

Comment:
I have executed this code on centos7 and got  .ssl1:102

Reply:
Fixed:
```
[root@compile-cent-7 mnt]# yum list installed|grep -i readline-devel|head -n1|awk '{print $2}'|awk -F "." '{print $1}'
6
```
[root@centos-6 vagrant]# yum list installed|grep -i readline-devel|head -n1|awk '{print $2}'|awk -F "." '{print $1}'
6
```